bash.exe.stackdump

```diff
@@ -0,0 +1,9 @@
+Stack trace:
+Frame        Function    Args
+000FFFFB440  0018006286E (0018028EC28, 0018026DE3E, 00000000000, 000FFFFA340)
+000FFFFB440  0018004846A (00000000000, 00000000000, 00000000000, 001004E2F5F)
+000FFFFB440  001800484A2 (0018028ECD9, 000FFFFB2F8, 00000000000, 00000000000)
+000FFFFB440  001800CC6BE (00000000000, 00000000000, 00000000000, 00000000000)
+000FFFFB440  001800CC7E5 (000FFFFB450, 00000000000, 00000000000, 00000000000)
+00000000000  001800CDDA5 (000FFFFB450, 00000000000, 00000000000, 00000000000)
+End of stack trace
```

Comment:
What's this?

Reply:
I have deleted the file, and re run the code, everything works fine